### DISCUSSION TITLE
[자료공유] Anaconda Navogator 사용 후기

### DISCUSSION THREAD

**Discussion by ghost** [4Moyede/HexatonClass-01#154] [04_자료공유]
> ## 문제 상황
다음 코드를 실행했습니다.
```
import pandas as pd
```
모듈이 없다는 오류 메시지가 출력! 열심히 구글링해보고 윈도우즈도 다시 설치하고 별 쇼를 다 했지만 해결 못함!

<br>

> ## 해결의 실마리
그러다가 <br>
![anagator01](https://user-images.githubusercontent.com/76792181/105204336-65b4ea80-5b87-11eb-9d7a-e57a9e08943c.png)
<br>여기서 Anaconda Navogator 를 발견! 혹시나 하는 마음에 들어가봤다.

<br>

> ## 사용 방법

여기 환경에 들어가서!<br>
![anagator02](https://user-images.githubusercontent.com/76792181/105204342-66e61780-5b87-11eb-81e4-a918b0e171ec.png)

<br> 본인의 가상환경을 고르고 내가 원하는 모듈이 설치되어 있는가를 살펴보면 된다!<br>
![anagator03](https://user-images.githubusercontent.com/76792181/105204347-677eae00-5b87-11eb-9d0b-9d6c69f08714.png)

<br> 저 같은 경우에는 galaxy (제 가상환경) 에 pandas, numpy, matplotlib 모두 깔려있지 않아서 발생한 문제였습니다.

<br>

> ## 끝맺음
저는 원하는대로 되지 않을 때 삭제하고 때려부수는 좋지 못한 습관이 있는데 잘 둘러보면 항상 해결책은 있기 마련이라는 것을 깨달았습니다. 다들 이 프로그램을 아실지는 모르겠지만, 구글링에 모듈이 없다고 아무리 검색해봐도 나오지 않았던 프로그램이기에 한번씩 들어가서 구경이라도 해보시면 도움이 될 듯 싶습니다.

**Comment by yoomambo** [comment]
@JaebeomHan 

```bash
conda list
```

command 입력해서 출력해보셨을까요? 사실 navigator를 쓰는 행위가 가상환경을 command 입력없이 GUI에서 보여주는 것이라,
제가 볼 때는 재범멘티님이 패키지 설치하실 때 base에 하셨거나, 아니면 가상환경에 설치하시고 activate를 안했거나 둘 중 하나 같긴합니다.

**Reply by ghost** [reply]
base에 설치한 것 같습니다.... 그래도 gui 환경에서 보니 더욱 확실하게 뭐가 문제였는지를 알 수 있었습니다.
